sdk/sdk/src/test/java/org/apache/carbondata/sdk/file/AvroCarbonWriterTest.java

```diff
@@ -47,26 +47,6 @@
 public class AvroCarbonWriterTest {
   private String path = "./AvroCarbonWriterSuiteWriteFiles";
 
-  @Before
```

Comment:
Remove unused imports from all test case files from this PR

Reply:
removed, please check